akvo/rsr/models/result.py

```diff
@@ -146,7 +146,7 @@ def child_projects(self):
         Return a dictionary of this result's child projects.
         """
         projects = {}
-        for result in Result.objects.filter(parent_result=self):
+        for result in Result.objects.filter(parent_result=self).select_related('project__title'):
```

Comment:
Oh, interesting, I thought it'd only fetch the title field from project. But if that's not the case, then it's pointless of course.